docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst

```diff
@@ -191,6 +191,9 @@ The following environment variables can be used to override the values in the JS
     export NVFLARE_XGB_PLUGIN_NAME=nvflare
     export NVFLARE_XGB_PLUGIN_PATH=/tmp/libnvflare.so
 
+.. note::
+   The plugin must be configured using environment variables for the simulator as it does not support resources.json.
```

Comment:
Thanks for this,

suggest a bit rephrase: "When running with the NVFlare simulator, the plugin must be configured using environment variables, as it does not support resources.json."

Reply:
Done.